README.md

```diff
@@ -1,13 +1,32 @@
-# Mongo API Project
+# City of birches
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The goals of this project were to practise modelling data in Mongoose, fetching items from a Mongo database using Mongoose and seeding a large amount of data to a database.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I wanted to use some data of interest to me. As I have studied landscape architecture I am familiar with trees. I found open data from municipality of Umeå with trees that are taken care of by the Park department. The original json that I could download from their open data site was a bit complicated with weird param naming and lots of nested objects and arrays. So I downloaded an excel file instead and changed it up a bit. Then I saved it as a CSV file and used a web app, csvjson.com to convert it to json. 
```

Comment:
That's awesome that you managed to change the file and make it from CSV to json. Good job! :D